langchain/package.json

```diff
@@ -812,7 +812,7 @@
     "jest": "^29.5.0",
     "jest-environment-node": "^29.6.4",
     "jsdom": "^22.1.0",
-    "llmonitor": "^0.5.4",
```

Comment:
You probably also want to bump the `peerDependency`?

Reply:
Oh sorry, didn't realize it was a `*` already - happy to leave it that way if you want, but will merge this as is.